source/routing/raptor_solution_reader.h

```diff
@@ -53,7 +53,24 @@ struct Dominates {
     }
 };
 
-typedef ParetoFront<Journey, Dominates> Solutions;
+struct ParetoFrontVisitor {
```

Comment:
`ParetoFrontLogger` might be a bit more precise ?

Reply:
Done https://github.com/CanalTP/navitia/pull/2937/commits/53cd072b2a5f6596daa177532ad871f67da15f12